server/world/provider.go

```diff
@@ -14,6 +16,11 @@ type Provider interface {
 	// SaveSettings saves the settings of a World.
 	SaveSettings(*Settings)
 
+	// LoadPlayerSpawnPosition loads the player spawn point if found, otherwise an error will be returned.
+	LoadPlayerSpawnPosition(uuid uuid.UUID) (pos mgl64.Vec3, exists bool, err error)
+	// SavePlayerSpawnPosition saves the player spawn point, in vanilla, this can be done with beds in the overworld
+	// and respawn anchors in the nether.
```

Comment:
```suggestion
	// SavePlayerSpawnPosition saves the player spawn point. In vanilla, this can be done with beds in the overworld
	// or respawn anchors in the nether.
```